### ISSUE TITLE
[ENH] Add a visualise or draw method for the graphical pipeline.

### ISSUE THREAD

**Issue by benHeid** [sktime/sktime#5218]
**Is your feature request related to a problem? Please describe.**
Currently, there is no possibility to visualise the structure of the graphical pipeline correctly. It would be nice to enable this.

**Describe the solution you'd like**
A possibility would be to reimplement the following solution from pywatts-pipeline: https://github.com/KIT-IAI/pywatts-pipeline/blob/main/pywatts_pipeline/utils/visualisation.py

Example Notebook: https://github.com/KIT-IAI/pywatts-pipeline/blob/main/examples/pipeline_visualization.ipynb

**Comment by fkiraly** [comment]
Yes - I noticed, it is also incorrect - or at least non-conforming with expectations - for the standard html visualization of the existing ones: https://github.com/sktime/sktime/issues/5233
